secure_software_development_fundamentals.md

```diff
@@ -3769,7 +3769,11 @@ In this course we have intentionally used more specific terms instead of DAST, i
 
 There are many other kinds of verification approaches, and many ways to combine them.
 
-A *penetration test* (aka *pen test*) simulates an attack on a system to try to break into (*penetrate*) the system. The people doing a penetration test are called penetration testers or a red team; they may be actively countered by a defensive team (also called a blue team). The point of a penetration test is to learn about weaknesses so they can be strengthened *before* a real attacker tries to attack the system.
+A *penetration test* (aka *pen test*) simulates an attack on a system to try to break into (*penetrate*) the system. The people doing a penetration test are called penetration testers. The point of a penetration test is to learn about weaknesses so they can be strengthened *before* a real adversary attacks the system.
```

Comment:
We could just talk about "penetration test" & "penetration testers".

I think in the next paragraph we should instead just say that a "red team" is a group of people who, as at least part of their duties, perform a penetration test. If we don't make claims about scope & duration, then we don't have to worry if everyone agrees (because I doubt they do).